### ISSUE TITLE
Processing lot questions

### ISSUE THREAD

**Issue by mhurcombe** [Integrity-Systems-Company/common_carcase_schema#3]
Confirming that the groupLotNumber might be used when a processing lot is split into multiple "lines" on any given kill date.
There is no expectation that body numbers are sequential within a processing lot or that all bodies between startBodyNumber and endBodyNumber are actually associated with the same processing lot. In other words, start and end body numbers are purely informational.

**Comment by cookeac** [comment]
Yes, my reading is that the key definition of a processingLot is the carcases included in that lot, and that start and end body numbers might be used by some processors, particularly where other carcase identifications (e.g. NLIS number) are not available for the species being processed. Some plants will split a Consignment into different "lines" and each line will be a processingLot, for others it is less clear (or more importantly, the schema needs to flex to cater for the various uses).

Have assigned @donwilson-rezare to check whether these fields are required or not in current implementations.

**Comment by donwilson-rezare** [comment]
Our definition of Carcase already includes a reference to the lot it belongs to.  So this handles non-sequential body numbers in a processing lot.
